web-components/src/components/icons/EditIcon.tsx

```diff
@@ -6,11 +6,16 @@ import { SvgLinearGradientProps } from './ProjectPageIcon';
 export default function EditIcon({
   sx,
   linearGradient,
+  disabled,
   ...props
 }: SvgLinearGradientProps): JSX.Element {
+  const randomId = Math.random().toString(36).substring(7);
+  const gradientId = disabled
+    ? `linear_edit_icon-${randomId}`
+    : 'linear_edit_icon';
   return (
     <SvgIcon
-      sx={[{ color: '#4FB573' }, ...sxToArray(sx)]}
+      sx={[{ color: disabled ? '#8F8F8F' : '#4FB573' }, ...sxToArray(sx)]}
```

Comment:
ditto
the initial implementation wasn't good, we should refactor it to used color var from our themes instead

Reply:
Same situation as in my previous comment https://github.com/regen-network/regen-web/pull/2475#discussion_r1773232462